### PR TITLE
Prevent pop-in of home screen logo from shifting the page; and delete unused graphics

### PR DESCRIPTION
**Pull request by advayDev1**
We hardcode the width, why not the height?

### CODE REVIEW

**Review by britannio — CHANGES_REQUESTED** [review]
The path for some of the deleted graphics is still present in `l10n.dart` however, they seem to be unused. Should they be removed?

**Comment by advayDev1** [comment]
> The path for some of the deleted graphics is still present in `l10n.dart` however, they seem to be unused. Should they be removed?

Hmm, yes they should be but I'm not sure how to auto-regen those files...

File: client/flutter/pubspec.yaml [hunk 1]
```diff
@@ -75,9 +75,7 @@ flutter:
     - assets/images/
     - assets/svg/
     - assets/animations/
-    - assets/news_press/
     - assets/onboarding/
-    - assets/travel_advice/
```
Comment (brunobowden) [inline client/flutter/pubspec.yaml:80]: These files are referenced in the arb file file, where they aren't removed. I'm expecting for them to be removed from the arb file(s) or to explain why it should be done any other way.

https://github.com/WorldHealthOrganization/app/blob/c02ffef9a0151dbe8248dcc745efa35b859b16b9/client/flutter/lib/l10n/intl_en_US.arb#L68

**Review by brunobowden — CHANGES_REQUESTED** [review]

**Comment by brunobowden** [comment]
@advayDev1 @britannio - I think they need to be removed from the `l10n/intl_en_US.arb` file... then I'm guessing they'll be removed from all the other generated dart files.

https://github.com/WorldHealthOrganization/app/blob/c02ffef9a0151dbe8248dcc745efa35b859b16b9/client/flutter/lib/l10n/intl_en_US.arb#L68

File: client/flutter/pubspec.yaml (hunk 1 above)
Comment (advayDev1) [reply client/flutter/pubspec.yaml:80]: fixed

**Comment by advayDev1** [comment]
Good call - removed all png references from loc files. PTAL @brunobowden @britannio

**Comment by advayDev1** [comment]
Please merge whenever one of you approves - Thanks.

**Review by theswerd — APPROVED** [review]

**Pull request merged**